docs/RSDKv3/Variables.md

```diff
@@ -0,0 +1,40 @@
+# RSDKv3 Variables
+
+# Script
+| Variable                                           | Description                                                            |
+| -------------------------------------------------- | ---------------------------------------------------------------------- |
+| **TempValue0-7**                                   | Temporary values used to store values during arithmetic or other similar operations. |
+| **ArrayPos0/1**                                    | Used to store an array of data. |
+| **Global**                                         | Can be used to use global variables as an array (Ex. Global[ArrayPos0] = 0). |
```

Comment:
All tables' Description column should have padding so that the ending `|` for each entry lines up. This applies to every table added in this PR.